packages/meditrak-server/src/apiV2/middleware/catchAsyncErrors.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+/**
+ * All routes will be wrapped with an error catcher that simply passes the error to the next()
+ * function, causing error handling middleware to be fired. Otherwise, async errors will be
+ * swallowed.
+ */
+export const catchAsyncErrors = routeHandler => (res, req, next) => {
```

Comment:
I see this is attached to all handlers, it would be nice to somehow generalise its usage... (No specific suggestion at this stage)

Reply:
Good point, but will keep that out of scope for now 🙏